### PR TITLE
HookTest 2

### PR DESCRIPTION
**Pull request by PonteIneptique**

### CODE REVIEW

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/csel-dev/210)*

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ python:
   - '3.5'
 
 install:
-  - pip3 install HookTest==0.2.2
+  - pip3 install HookTest==1.0.0
 
-script: hooktest --scheme epidoc --workers 3 --verbose --travis --countword --allowfailure ./ --from_travis_to_hook "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/csel-dev"
+script: hooktest ./ --scheme epidoc --workers 3 --verbose 10 --manifest --countword --allowfailure --console table --hookUI "http://ci.perseids.org/api/hook/v2.0/user/repositories/OpenGreekAndLatin/csel-dev"
```
Comment (sonofmun) [inline .travis.yml]: You should take --allowfailure out of the command line since this is a 100% repo.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/csel-dev/219)*

**Comment by sonofmun** [comment]
Hmm. Should the verbosity be set lower for a repo like this? I am just thinking that probably all that we want to know with each test here is what fails, since we expect everything to pass. What verbosity level would that be? Maybe 7?

**Comment by capitains-hook** [comment]
*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/csel-dev/222)*

**Pull request merged**